Atari-Game-Player/materializer/replay_buffer_materializer.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Type, Union, List
+import pickle
+import os 
+DEFAULT_FILENAME = "PyEnvironment"
+from zenml.io import fileio
+from zenml.materializers.base_materializer import BaseMaterializer
+from DQN.model import ReplayBuffer
+
+class ReplayBufferMaterializer(BaseMaterializer):
+    ASSOCIATED_TYPES = [ReplayBuffer]
+
+    def handle_input(
+        self, data_type: Type[Any]
+    ) -> Union[ReplayBuffer, ReplayBuffer]:
+        """Reads a base sklearn label encoder from a pickle file."""

```

Comment:
```suggestion
        """Reads a RelayBuffer from a pickle file."""

```

Reply:
Small typo there, ReplayBuffer, not RelayBuffer